dirsrvtests/tests/suites/config/config_test.py

```diff
@@ -551,6 +551,26 @@ def test_require_internal_index(topo):
         user.delete()
 
 
+@pytest.mark.bz1897248
+@pytest.mark.ds4315
+@pytest.mark.skipif(ds_is_older('1.4.3.16'), reason="This config setting exists in 1.4.3.16 and higher")
+def test_nagle_default_value(topo):
+    """Test that nsslapd-nagle attribute is off by default
```

Comment:
Probably, it makes more sense to put it at the beginning of the test suite as it checks for a default value.